tests/loggers/test_all.py

```diff
@@ -79,9 +79,16 @@ def log_metrics(self, metrics, step):
     trainer.test()
 
     log_metric_names = [(s, sorted(m.keys())) for s, m in logger.history]
-    assert log_metric_names == [(0, ['epoch', 'val_acc', 'val_loss']),
-                                (0, ['epoch', 'train_some_val']),
-                                (1, ['epoch', 'test_acc', 'test_loss'])]
+    if logger_class == TensorBoardLogger:
+        assert log_metric_names == [(0, ['hp_metric']),
+                                    (0, ['epoch', 'val_acc', 'val_loss']),
+                                    (0, ['epoch', 'train_some_val']),
+                                    (0, ['hp_metric']),
```

Comment:
If hp_metric is here twice on purpose, can you write a comment why?

Reply:
because test calls log_hyperparams as well so it is logged twice (fit + test)

Reply:
ok